### PR TITLE
docker-selenium: unset setuid bit

### PR DESCRIPTION
**Pull request by imjasonh**
#22566

### CODE REVIEW

**Comment by imjasonh** [comment]
The build now completes successfully without the linter warning (which can soon become a required check)

**Review by dakaneye — DISMISSED** [review]
LGTM, docker-selenium's image user shouldn't need access to this file directly anyways (just to run supervisord, managing a selenium hub or something)

File: docker-selenium.yaml [hunk 1]
```diff
@@ -177,6 +177,9 @@ subpackages:
           mv ${{targets.destdir}}/etc/supervisord.conf ${{targets.subpkgdir}}/etc
           mv ${{targets.destdir}}/etc/supervisor ${{targets.subpkgdir}}/etc
 
+          # Unset setuid bit
+          chmod 00775 ${{targets.subpkgdir}}/etc/supervisord.conf
```
[the remark this answers is not in the document]
Comment (imjasonh) [reply docker-selenium.yaml:181]: I copied this directly from https://serverfault.com/a/238964 -- the extra `0` is to unset the `setuid` bit apparently.

[the remark this answers is not in the document]
Comment (smoser) [reply docker-selenium.yaml:181]: That is new information to me.  it feels like maybe this is an embarrasing moment where i'll be shown to be wrong, but ....

```
root@n1:~# id
uid=0(root) gid=0(root) groups=0(root)
root@n1:~# touch foo
root@n1:~# chmod 7444 foo
root@n1:~# stat foo > stat.7444
root@n1:~# chmod 0444 foo
root@n1:~# stat foo > stat.0444
root@n1:~# diff -u stat.7444 stat.0444
--- stat.7444	2024-06-25 16:39:22.025859953 +0000
+++ stat.0444	2024-06-25 16:39:32.109727580 +0000
@@ -1,8 +1,8 @@
   File: foo
   Size: 0         	Blocks: 0          IO Block: 4096   regular empty file
 Device: 253,3	Inode: 414851      Links: 1
-Access: (7444/-r-Sr-Sr-T)  Uid: (    0/    root)   Gid: (    0/    root)
+Access: (0444/-r--r--r--)  Uid: (    0/    root)   Gid: (    0/    root)
 Access: 2024-06-25 16:38:52.742244681 +0000
 Modify: 2024-06-25 16:38:52.742244681 +0000
-Change: 2024-06-25 16:39:02.430117348 +0000
+Change: 2024-06-25 16:39:25.997807806 +0000
  Birth: 2024-06-25 16:38:52.742244681 +0000
```

that shows a single 0 `0444` seems to clear things fine.

[the remark this answers is not in the document]
Comment (smoser) [reply docker-selenium.yaml:181]: well, there you have it. it _does_ matter for directories.

```
root@n1:~# rm -Rf bar-dir/
root@n1:~# mkdir bar-dir
root@n1:~# chmod 7777 bar-dir
root@n1:~# stat bar-dir > stat-dir.7777
root@n1:~# chmod 0777 bar-dir
root@n1:~# stat bar-dir > stat-dir.0777
root@n1:~# chmod 00777 bar-dir
root@n1:~# stat bar-dir > stat-dir.00777
root@n1:~# diff -u stat-dir.7777 stat-dir.0777
--- stat-dir.7777	2024-06-25 16:44:33.593790881 +0000
+++ stat-dir.0777	2024-06-25 16:44:48.817592961 +0000
@@ -1,8 +1,8 @@
   File: bar-dir
   Size: 4096      	Blocks: 8          IO Block: 4096   directory
 Device: 253,3	Inode: 670772      Links: 2
-Access: (7777/drwsrwsrwt)  Uid: (    0/    root)   Gid: (    0/    root)
+Access: (6777/drwsrwsrwx)  Uid: (    0/    root)   Gid: (    0/    root)
 Access: 2024-06-25 16:44:12.714062442 +0000
 Modify: 2024-06-25 16:44:12.714062442 +0000
-Change: 2024-06-25 16:44:21.937942460 +0000
+Change: 2024-06-25 16:44:39.481714326 +0000
  Birth: 2024-06-25 16:44:12.714062442 +0000
root@n1:~# diff -u stat-dir.0777 stat-dir.00777
--- stat-dir.0777	2024-06-25 16:44:48.817592961 +0000
+++ stat-dir.00777	2024-06-25 16:45:02.589413977 +0000
@@ -1,8 +1,8 @@
   File: bar-dir
   Size: 4096      	Blocks: 8          IO Block: 4096   directory
 Device: 253,3	Inode: 670772      Links: 2
-Access: (6777/drwsrwsrwx)  Uid: (    0/    root)   Gid: (    0/    root)
+Access: (0777/drwxrwxrwx)  Uid: (    0/    root)   Gid: (    0/    root)
 Access: 2024-06-25 16:44:12.714062442 +0000
 Modify: 2024-06-25 16:44:12.714062442 +0000
-Change: 2024-06-25 16:44:39.481714326 +0000
+Change: 2024-06-25 16:44:54.121524023 +0000
  Birth: 2024-06-25 16:44:12.714062442 +0000
```

**Review by smoser — CHANGES_REQUESTED** [review]
could you put a comment pointing to that server fault ? otherwise, future smoser is going to say "what is that!"

**Review by smoser — APPROVED** [review]
thanks.

**Pull request merged**